remove check apr

this patch removes the check of apr which fails the build

Signed-off-by: Rico Bachmann <bachmann-ahF9wiJNQgNBDgjK7y7TUQ@public.gmane.org>
---
v1: disable the check for APU_FIND_APR because the check only looks into folders
named apr, and our source-tree is called apr-versionnumber (e.g. apr-1.4.6)

diff -rupN apr-util-1.4.1/configure.in apr-util-1.4.1-fix/configure.in
--- apr-util-1.4.1/configure.in	2009-12-17 20:15:19.000000000 +0100
+++ apr-util-1.4.1-fix/configure.in	2012-04-24 10:10:32.000000000 +0200
@@ -99,7 +99,7 @@ fi
 dnl
 dnl Find the APR includes directory and (possibly) the source (base) dir.
 dnl
-APU_FIND_APR
+dnl APU_FIND_APR
 
 dnl
 dnl even though we use apr_rules.mk for building apr-util, we need
